src/maui/SSW.Rewards.Mobile/SSW.Rewards.Mobile/Pages/QuizDetailsPage.xaml

```diff
@@ -69,22 +72,22 @@
                                   CurrentItemChangedCommand="{Binding CurrentQuestionChangedCommand}"
                                   x:Name="QuestionsCarousel">
                         <CarouselView.ItemTemplate>
-                            <DataTemplate>
+                            <DataTemplate x:DataType="viewModels:QuizQuestionViewModel">
                                 <ScrollView HeightRequest="700">
                                     <StackLayout BindableLayout.ItemsSource="{Binding MyAnswers}"
                                                  RadioButtonGroup.GroupName="{Binding QuestionId}"
                                                  HorizontalOptions="FillAndExpand"
                                                  Padding="20"
                                                  Spacing="40">
                                         <BindableLayout.ItemTemplate>
-                                            <DataTemplate>
-                                                <RadioButton Value="{Binding QestionAnswerId}"
+                                            <DataTemplate x:DataType="viewModels:QuizAnswerViewModel">
+                                                <RadioButton Value="{Binding QuestionAnswerId}"
```

Comment:
By applying `x:DataType` we picked up a spelling mistake. It was `QestionAnswerId` while should be `QuestionAnswerId`. Cool 😎